index.js

```diff
@@ -0,0 +1,2 @@
+require('./dist/json-formatter');
+module.exports = 'jsonFormatter';
```

Comment:
@mohsen1 Sorry for the late response.

In short: Yes, you have to export the string with angular module name.

There is a convention which angular team (and not only) uses for their npm modules.
For example look at https://www.npmjs.com/package/angular-animate
```
angular.module('myApp', [require('angular-animate')]);
```

Isn't it handy? :)